src/helpers.ts

```diff
@@ -202,6 +202,13 @@ export function editBackendTopLevelJson(appName: string, targetDir: string) {
       "dev:frontend"
     ] = `yarn workspace @${appName}/frontend start:dev`
     json.scripts["dev:backend"] = `yarn workspace @${appName}/backend start:dev`
+    json.scripts["start:frontend"] = `yarn workspace @${appName}/frontend start`
+    json.scripts["start:backend"] = `yarn workspace @${appName}/backend start`
+    json.scripts["build"] = `yarn workspaces run build`
+    json.scripts[
+      "db:run:migrations"
+    ] = `yarn workspace @${appName}/backend db:migration:run:prod`
```

Comment:
Yeah...I was thinking about this. I believe we need to build the backend in order to build the frontend since we need to generate the graphql schema, but not the frontend to build the backend. That's true right, @hallettj? I'm not going to do this now, but I imagine it's an optimization we should make in the near future.

Reply:
`schema.graphql` is checked into version control, so I don't think there is any build dependency either way